src/include/cpp-btree/btree_container.h

```diff
@@ -40,10 +35,14 @@ class btree_container {
   // `key_arg<K>` evaluates to `K` when the functors are transparent and to
   // `key_type` otherwise. It permits template argument deduction on `K` for the
   // transparent case.
+  template <class Compare>
+  using is_transparent_t = typename Compare::is_transparent;
```

Comment:
Do we really need this? I'm not 100% sure but it looks the only things with `is_transparent` defined were `StringBtreeDefaultLess` and `StringBtreeDefaultGreater`. We dropped both of them.

Reply:
`is_transparent` is a part of C++14 standard. and is respected by associate containers for heterogeneous comparison lookup. see https://en.cppreference.com/w/cpp/container/set/count . btree_container is trying to follow this convention for supporting this feature. see also http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2013/n3657.htm.